src/arweave/ArweaveProxy.ts

```diff
@@ -39,8 +40,8 @@ export default class ArweaveProxy  {
   }
 
   async getBalance(): Promise<number> {
```

Comment:
Pls, let's wrap this one in try catch in case Arweave is down

Reply:
so if someone has too low balance to start a redstone Node - we won't allow him to run a Node, but if we can't determine what's the balance - redstone Node should be started? uhm, we need to talk :-)

Reply:
We should continue to operate even if Arweave is down. Without catching the error the node will do down on the first error received from Arweave.